meinberlin/apps/moderatorremark/api.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import mixins
+from rest_framework import viewsets
+
+from adhocracy4.api.mixins import AllowPUTAsCreateMixin
+from adhocracy4.api.permissions import IsModerator
+
+from .models import ModeratorRemark
+from .serializers import ModeratorRemarkSerializer
+
+
+class ModeratorRemarkViewSet(AllowPUTAsCreateMixin,
+                             mixins.UpdateModelMixin,
+                             viewsets.GenericViewSet):
```

Comment:
provide a `retrieve()` method or try adding the `mixins.RetrieveModelMixin` from drf to allow for `GET`ting objects